### PR TITLE
Added github-workflow for comparing translation file

### LINKED ISSUE #1185: GitHub Action Required for Translation Files
We need to create a GitHub action that during the PR process checks all the translation files to make sure that there are translations in all the non-default languages for each English default language entry.

As part of the PR for this issue, all the translations must be correctly present

- **NOTE:** Please watch this video on how to proceed.
[Click me to start video](https://www.youtube.com/watch?v=GlqUm1UO-xM)

### PR DESCRIPTION
**Pull request by 1010varun**
**What kind of change does this PR introduce?**

*Added GitHub workflow for comparing translation files*

**Issue Number:**

Fixes #1185 

**Did you add tests for your changes?**

**Snapshots/Videos:**

image

**If relevant, did you update the documentation?**

**Summary**

In this PR, I have introduced a GitHub workflow and a Python script that will check all the translation files and reflect the comparisons between different language files with English one.

**Does this PR introduce a breaking change?**

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

**Comment by palisadoes** [comment]
Please fix the new failing translation test

**Comment by 1010varun** [comment]
> Please fix the new failing translation test

Hey,
@palisadoes, sir I have all the failing translation files. 
@noman2002 kindly review it once.

**Review by palisadoes — CHANGES_REQUESTED** [review]
1. Please use the same file naming convention as this PR for consistency
- https://github.com/PalisadoesFoundation/talawa/pull/2284
1. The python file is not python `black`, `pycodestyle`, `pylint` and `pydocstyle` compliant. Please fix this.

**Comment by 1010varun** [comment]
@palisadoes, sir I have updated the naming of the file acc. to this PR, kindly review it once.

File: .github/workflows/compare_translations.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
+"""Script to encourage more efficient coding practices.
+Methodology:
+
```
Comment (palisadoes) [inline .github/workflows/compare_translations.py:3]: What changes did you make to this file, that's based on the `Talawa` translations check script, and why?

**Review by palisadoes — CHANGES_REQUESTED** [review]
See comment

File: .github/workflows/compare_translations.py (hunk 1 above)
Comment (1010varun) [reply .github/workflows/compare_translations.py:3]: sir you have asked to make this file python black, pycodestyle, pylint and pydocstyle compliant. So, I have made some changes based on the Talawa translations check script.

Comment (palisadoes) [reply .github/workflows/compare_translations.py:3]: 1. Did you make any changes to the logic? 
1. Will this script perform differently in any given scenario than the other one and if so, why?

Comment (1010varun) [reply .github/workflows/compare_translations.py:3]: Sir, as both issues are similar, therefore I have taken a reference from that script. Furthermore, this script will work as same as that one and will produce the same result as that one.
If you want I can make changes in this.

**Review by palisadoes — APPROVED** [review]

**Pull request merged**